tianshou/data/collector.py

```diff
@@ -51,13 +52,14 @@ class Collector(object):
     def __init__(
         self,
         policy: BasePolicy,
-        env: Union[gym.Env, BaseVectorEnv],
+        env: Union[gym.Env, BaseVectorEnv, pettingzoo.AECEnv],
```

Comment:
Second thought: since you have used BaseVectorEnv to execute PettingzooEnv, add this special check is useless?

Reply:
Done 👍